### ISSUE TITLE
Client Allocation Request for: Jeweltours

### ISSUE THREAD

**Issue by ayeishanasheena** [filecoin-project/filecoin-plus-client-onboarding#735]
---

# Client Allocation Questions 

## Core Information
- Name: Jeweltours
- Website / Social Media: https://www.jeweltours.com/
- Region: Asia excl. Greater China
- DataCap Requested: 5TiB
- Addresses to be Notarized: f1uyr5lvule563sf4kmi33w7v6ybpsjhabhsn7qda
- Notary Requested: IreneYoung

@ayeishanasheena Please subscribe to notifications for this Issue to be aware of updates. Notaries may request additional information on the Issue.

**Comment by ayeishanasheena** [comment]
Basic Info

1.What's your name and your position in your company?
Hello sir,my name is Nguyễn Thanh Tùng. My position in the company is network maintenance and management.

2.Please introduce yourself and your team briefly.
I am from Vietnam and graduated from Hue University. Major is information technology.
We have a team of 16 people. All of the team is young. The people on our team are from Vietnam, the United States and China .

3.For your organization, Please describe its business, its history since inception, registration country, scale, named executive officers, relevant projects, industry reputation, etc.
Jewel Tours was founded in 2005 by Mr. Thanh Tien Dinh. We operate tours to Vietnam, Cambodia, Laos, Thailand, Myanmar and Asian destinations mentioned on our website. All destinations chosen by the owners of Jewel Tours focus on real vacations, treat customers as individuals, and support sustainable tourism.

4.Please provide your company's business registration license. And show us some of your web pages, Github, social media accounts such as Twitter, Facebook, Naver blog, WeChat official account, etc. Please include as many relevant details to provide contextual information (with external links). If you have any presentation documents about your business, please provide them to us.
The daily work of our team is to maintain the normal operation of the website and develop some new projects and functions at the same time. Recently devoted to new block information technology.
Website: https://www.jeweltours.com
Media link:https://www.facebook.com/people/Jewels-Tours/100063353641882/

5.What types of data do you plan to store with the DataCap and how many do you need? Could you please send us some of your sample data?
We plan to use DataCap to store some internal company files. The amount of 5~10 Tib is needed. Sample data I will send you via E-,mail with the business license  later.

6.Regarding the data you intend to store, What’s the original source of those data, and do you have any proof of the ownership or authorization?
The data we intend to store are all the photos and video clips that we record every part of their trip for tourists. Some customers will agree to share a series of their photos.


Track Record

1.How did you know about Filecoin? Please feel free to describe your understanding of Filecoin storage and the Filecoin Plus program.
We got to know it through the internet. About the Filecoin storage,we think it's great.Significantly reduces the cost of data storage.Regarding the Filecoin Plus program, we believe it is a revolutionary project and a symbol that Filecoin is coming great.

2.Has your organization obtained DataCap before?
None.

3.Are there any disputes open against you from your previous DataCap allocations? Please describe any of your honors or disputes in the field of Filecoin.
None.


Allocation Plan

1.Please describe your plan to use the DataCap as detailed as possible? (including use case, amount, timespan, geography, miner, partner, etc.)
We intend to use DataCap to store some videos and photos, we have 40T of data, this time we intend to request 5T

2.Do you plan to record your allocation decisions and execution details in public? And how to realize it?
What's your plan to make sure the DataCap you have obtained will be used to its best advantage?
We will make our distribution records public in Github.
We will continue to watch the use of DataCap to prevent incorrect use.

3.Do you agree to use the DataCap to only store data that abide by local regulations and is in compliance with the recipient miner’s terms of service?
Yes I agree.

**Comment by IreneYoung** [comment]
Thanks @ayeishanasheena ! Please send your business license and sample data to the email address: contact@12ships.com

**Comment by ayeishanasheena** [comment]
Hi,IreneYoung I've sent you the email

**Comment by IreneYoung** [comment]
On the part of your answers to the Basic Info Q5 and Q6, we are concerned about the privacy issues. 
To our knowledge, whatever the internal company files or the photos and video clips that you record for tourists, they should be private data for the owners. Are you sure these data could be stored publicly on Filecoin without privacy concerns? You should know that any data stored on Filecoin can be retrieved so that it must be public data.

**Comment by ayeishanasheena** [comment]
Yes, it can be made public.

**Comment by IreneYoung** [comment]
Hey @ayeishanasheena, the initial score of your DataCap request is 3, and 5TiB DataCap will be granted to you. Congratulate! 
See more details here: https://docs.google.com/spreadsheets/d/1UdKvQWIKkKnZHlZbNFbxbAMSlSHNgji7G2msQva7f88/edit?usp=sharing

Please double-check and confirm your address: f1uyr5lvule563sf4kmi33w7v6ybpsjhabhsn7qda

**Comment by ayeishanasheena** [comment]
Yes,I confirm this address: f1uyr5lvule563sf4kmi33w7v6ybpsjhabhsn7qda

**Comment by SAONetwork01** [comment]
@IreneYoung @dkkapur Hello, I have submitted a request for DataCap (https://github.com/filecoin-project/filecoin-plus-client-onboarding/issues/897) and I reply to you by email. Please check that. Thanks!

**Comment by mr-spaghetti-code** [comment]
Hi,

It looks like you received 5TiBs of DataCap to date but have spent less than 20% of it so far.

We would love to understand if there's anything holding you back. We are working hard to make the data onboarding process easier for clients like you and your feedback is very valuable. If you have a moment, please fill in this survey: https://forms.gle/s6AuTXZPZSMokscLA

If you have any feedback or would like to consult with an expert, please let me know. 

Thanks,

João Fiadeiro
Product Manager, Large Data Client Onboarding
Protocol Labs
